exercises/practice/bottle-song/.meta/example.lua

```diff
@@ -0,0 +1,52 @@
+local BottleSong = {}
+
+local GENERIC_VERSE = table.concat({
+  "N green bottles hanging on the wall,\n",
+  "N green bottles hanging on the wall,\n",
+  "And if one green bottle should accidentally fall,\n",
+  "There'll be M green bottles hanging on the wall.\n"
+})
+
+local words = {
+  'one',
+  'two',
+  'three',
+  'four',
+  'five',
+  'six',
+  'seven',
+  'eight',
+  'nine',
+  'ten'
+}
+words[0] = 'no'
```

Comment:
```suggestion
local words = {
  [0] = 'no',
  'one',
  'two',
  'three',
  'four',
  'five',
  'six',
  'seven',
  'eight',
  'nine',
  'ten'
}
```

I think this will work